index.js

```diff
@@ -731,7 +731,7 @@ _awaitWriteFinish(path, threshold, event, awfEmit) {
 }
 
 _getGlobIgnored() {
-  return Array.from(this._ignoredPaths.values());
+  return [...this._ignoredPaths.values()];
```

Comment:
How fast is this when compared to older way?

Reply:
I didn't make any benchmarks. Do you have a script we can use for benchmarking?

Reply:
Just tested online, no real difference